samples/android-integration/permissions/src/main/java/com/badoo/ribs/samples/permissions/rib/parent/PermissionsSampleParentView.kt

```diff
@@ -32,5 +32,10 @@ class PermissionsSampleParentViewImpl private constructor(
         }
     }
 
-    override fun getParentViewForSubtree(subtreeOf: Node<*>): ViewGroup = childContainer
+    override fun getParentViewForSubtree(subtreeOf: Node<*>): ViewGroup =
+        when (subtreeOf.parent?.children?.indexOf(subtreeOf) ?: -1) {
+            0 -> child1Container
+            1 -> child2Container
+            else -> super.getParentViewForSubtree(subtreeOf)
```

Comment:
This is super hacky :) Normally we should simply rely on incoming interface, but yes, now they are the same. Let's try to figure out something for this.